client/js/widgets/widget.js

```diff
@@ -823,21 +823,15 @@ export class Widget extends StateManaged {
               return false;
             if(a.type != 'all' && (w.get('type') != a.type && (a.type != 'card' || w.get('type') != 'pile')))
               return false;
-            if(a.relation === '<')
-              return w.get(a.property) < a.value;
-            else if(a.relation === '<=')
-              return w.get(a.property) <= a.value;
-            else if(a.relation === '!=')
-              return w.get(a.property) != a.value;
-            else if(a.relation === '>=')
-              return w.get(a.property) >= a.value;
-            else if(a.relation === '>')
-              return w.get(a.property) > a.value;
-            else if(a.relation === 'in' && Array.isArray(a.value))
-              return a.value.indexOf(w.get(a.property)) != -1;
-            if(a.relation != '==')
-              problems.push(`Warning: Relation ${a.relation} interpreted as ==.`);
-            return w.get(a.property) === a.value;
+            if(a.relation == '==') {
```

Comment:
This is making `==` impossible to use.

Reply:
If you look carefully in current code, it is already impossible to use. I'm just making it obvious.

Reply:
I see we've been testing `===` this whole time and calling it `==`